python/cudf/cudf/tests/test_indexing.py

```diff
@@ -117,6 +117,16 @@ def test_series_indexing(i1, i2, i3):
             assert series[i] == a1[i]
 
 
+def test_series_indexing_large_size():
+    n_elem = 100_000
+    gsr = cudf.Series(np.ones(n_elem))
```

Comment:
This will generate a `100_000` size host array which sounds expensive. Could we reduce the size or use cupy instead?

Reply:
Can't reduce size, can use cupy.